### ISSUE TITLE
Kitematic Not working after Docker update

### ISSUE THREAD

**Issue by hiteshkumar2187** [docker/kitematic#3795]
### Expected behavior
Kitematic should work after docker updated
### Actual behavior
Kitematic throws an error stating "There seems to have been an expected error with Kitematic"
### Information about the Issue
Current Docker version : Version 18.03.1-ce-mac64 (24245)
Compose version : 1.21.0
Machine: 0.14.0
Notary: 0.6.1
OS - macOS High Sierra , Version 10.13.1 (17B1003)

### Steps to reproduce the behavior

  1. Install Docker,
  2. Install Kitematic
  3. Enable Docker updates, On update request, Update docker.

<img width="560" alt="screen shot 2018-04-30 at 5 17 45 pm" src="https://user-images.githubusercontent.com/4910137/39425757-74ad40ea-4c9a-11e8-9ae7-d58ec696d657.png">
<img width="1158" alt="screen shot 2018-04-30 at 5 17 33 pm" src="https://user-images.githubusercontent.com/4910137/39425758-74d6a610-4c9a-11e8-93a5-3cfb50b6da20.png">

**Comment by FrenchBen** [comment]
It's possible that some of the settings may have caused this. If you look, to the bottom right of the red container image, there's a `reset` button. 
Could you give that a try and see if it fixes your issue?

**Comment by jdharri** [comment]
I'm having the same issue on windows

**Comment by jgschmitz** [comment]
Yeah this happened to me also after upgrading to Mojave I had to uninstall everything docker related and start over for it to work again - (nothing local on my mac is production)

**Comment by hiteshkumar2187** [comment]
> It's possible that some of the settings may have caused this. If you look, to the bottom right of the red container image, there's a `reset` button.
> Could you give that a try and see if it fixes your issue?

I tried resetting it as well. At the end, I have to uninstall everything related to docker and start over again to make it work.
